### PR TITLE
Correct the indentation (#2243)

### PR DESCRIPTION
**Pull request by Utkarsh-Mishra-CIC**
**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by Utkarsh-Mishra-CIC** [comment]
should i enter those points in commit description? And please correct me if i'm wrong, that description is all the text after `#`, right?

**Comment by newbazz** [comment]
nope, not like that, there should be three parts of commit,
Read about them on 
http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html

**Comment by Utkarsh-Mishra-CIC** [comment]
I think now it's fine, or should i add all the 3 lines which are changed? but then description would be more than 50 words.

**Comment by newbazz** [comment]
`Closes <issue_link> should be in a new line.

File: bears/c_languages/CPPLintBear.py [hunk 1]
```diff
@@ -28,9 +28,12 @@ def create_arguments(filename, file, config_file,
                          cpplint_ignore: typed_list(str)=(),
                          cpplint_include: typed_list(str)=()):
         """
-        :param max_line_length: Maximum number of characters for a line.
-        :param cpplint_ignore:  List of checkers to ignore.
-        :param cpplint_include: List of checkers to explicitly enable.
+        :param max_line_length:
+             Maximum number of characters for a line.
```
Comment (khanchi97) [inline bears/c_languages/CPPLintBear.py]: Should be indented by 4 spaces ;)

Comment (Utkarsh-Mishra-CIC) [reply bears/c_languages/CPPLintBear.py]: is it fine now?

**Comment by yash-nisar** [comment]
Looks good !

**Comment by yash-nisar** [comment]
ack ae8134e160ade1b32ae2af3eb76a8c11f0f6a7bf

**Comment by Makman2** [comment]
ack ae8134e160ade1b32ae2af3eb76a8c11f0f6a7bf

**Comment by Makman2** [comment]
@gitmate-bot rebase

**Comment by Makman2** [comment]
@gitmate-bot ff

**Pull request merged**